### DISCUSSION TITLE
Rofi like Tab Switching for Open Notes

### DISCUSSION THREAD

**Discussion by protean-onion** [TriliumNext/Trilium#3730] [Ideas]
Hello,

Sometimes, I find myself having multiple files open simultaneously. To navigate to the desired file, I have to use my cursor or to keep pressing `Ctrl + Tab` till I arrive at the desired file. While this is alright, I think it would be great if we could pull up a menu of open files with the press of a button combination and then just select and open a desired file. This functionality has been implemented for open windows in [`rofi`](https://github.com/davatorium/rofi).

If this is a plausible idea, would you implement it? If yes, how long would it take?

Thanks for your time.

**Comment by zadam** [comment]
Hi, I created a feature request since that's a more proper place for this: https://github.com/zadam/trilium/issues/3735

No promises about implementation, though.

**Reply by protean-onion** [reply]
That's alright. Thanks for taking notice of my request. I still look forward to it.
